mos-platform/geos-cbm/clang.cfg

```diff
@@ -0,0 +1,8 @@
+-Os
+-flto
+-fnonreentrant
+-mllvm -inline-threshold=0
```

Comment:
Is there a particular reason for setting `inline-threshold=0`?

Reply:
It was inlining way to much. Also Code from overlay Module. So I tried that.

Reply:
I prefer to use `-Os -flto` and let the compiler decide what to inline. Or did that produce (too) large binaries?